### DISCUSSION TITLE
Roadmap?

### DISCUSSION THREAD

**Discussion by Vitalium** [aminalaee/mongox#92] [Q&A]
Hello!

Do you have a roadmap for features implementation?

For example, support for models referencing, inheritance, before/after events, aggregation, projections etc?

Thank you!

**Comment by nistvan86** [comment]
Bump, I'm also interested in aggregations and projections.

**Reply by aminalaee** [reply]
Hi @Vitalium  and @nistvan86 

TBH I think right now there are a few existing Python ODM projects which look great. My idea was the same thing but wit nice and different API, which I saw there was not much use case for it.

If I see some activity in the repo like reporting issues, new features and PRs I will try to dedicate more time to it.

**Reply by nistvan86** [reply]
Thank you. I've just found Beanie for a similar use case which seems to support what I need. https://beanie-odm.dev/
Leaving it here for those who might stumble upon to this discussion.
